cmd/notary/tuf.go

```diff
@@ -889,7 +889,13 @@ func tokenAuth(trustServerURL string, baseTransport *http.Transport, gun data.GU
 	if endpoint.Scheme == "" {
 		return nil, fmt.Errorf("Trust server url has to be in the form of http(s)://URL:PORT. Got: %s", trustServerURL)
 	}
-	subPath, err := url.Parse("v2/")
+	var p string
+	if strings.HasSuffix(endpoint.Path, "/") {
```

Comment:
I think you can just use `filepath.Join` here, which would take care of de-duping the `/` for you (and even de-duping extra `/` as well:  https://play.golang.org/p/ALPqJzKDCq)